Sources/BraintreeDropIn/Custom Views/BTUIPaymentMethodCollectionViewCell.m

```diff
@@ -35,7 +35,7 @@ - (instancetype)initWithFrame:(CGRect)frame
         [self.contentView addSubview:self.paymentOptionCardView];
 
         self.titleLabel = [[UILabel alloc] init];
-        [BTUIKAppearance styleSmallLabelBoldPrimary:self.titleLabel];
+        [BTUIKAppearance styleSmallLabelPrimary:self.titleLabel];
```

Comment:
This was the only place that the `styleSmallLabelBoldPrimary` method was used, and Apple doesn't provide a bold dynamic font style that small enough to match what we had previously. Opted to make it not bold instead. (This is the title -- e.g., PayPal, Visa, MasterCard -- that appears under the icon in the vaulted payment method collection view.)